### PR TITLE
Document ScamGuard Commands

### PR DESCRIPTION
**Pull request by user4752**
initial pass at collating and collecting all ScamGuard commands into one file and document purpose and parameters for #46 

Formatting is tentative as markdown is fairly limited as to what it can create for this style of documentation.

Note: documenting these commands highlights several commands which are near identical and could possibly be merged for deduplication and made as aliases.

### CODE REVIEW

File: documentation.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+| Cooldown | once every 3 seconds |
+
+| Arguments | Type |
+| --- | --- |
+| userid | discord userid to check |
+
+
+## Global
+> These commands can be run on any server where ScamGuard is joined
+
```
Comment (SocksTheWolf) [inline documentation.md:127]: Might want to preface that all of these commands require a `/scamguard` before them.

File: documentation.md [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+### `check userid`
+|     |     |
+| --- | --- | 
+| Description | Checks to see if a userid is banned |
+| Permissions | can ban members |
+
+| Arguments | Type |
+| --- | --- |
+| userid | discord userid to check |
+
```
Comment (SocksTheWolf) [inline documentation.md:137]: Missing the commands for reporting.

File: documentation.md (hunk 1 above)
Comment (SocksTheWolf) [reply documentation.md:127]: And currently only these commands

File: documentation.md [hunk 3]
```diff
@@ -0,0 +1,155 @@
+#  ScamGuard Command Documentation
```
Comment (SocksTheWolf) [inline documentation.md:1]: I'm wondering if we want to put this into the wiki first, I forgot to actually turn on the wiki setting on github.

**Review by SocksTheWolf — CHANGES_REQUESTED** [review]
Thanks for the great work on this so far! Let's actually make this a wiki page instead.

File: documentation.md (hunk 1 above)
Comment (user4752) [reply documentation.md:124]: I had a header at the top of the file, but i misread the code and thought all commands were /scamguard activated. clarification has been moved and reworded to this section now.

File: documentation.md (hunk 2 above)
Comment (user4752) [reply documentation.md:134]: I genuinely have no idea how i missed these. updated.

File: documentation.md (hunk 3 above)
Comment (user4752) [reply documentation.md:1]: verbiage changed.

**Comment by user4752** [comment]
There is currently no GitHub way to make a pull request or submit changes to a wiki outside of direct access to the repository via the website itself. Best I can figure is if documentation is approved, the request should be closed and the contents manually copied to a wiki page by yourself.

**Comment by SocksTheWolf** [comment]
The wiki should be open for contributors to add to, but I went ahead and made the page. Closing, thank you for your contribution!

**Pull request closed without merging**